Resources/NetHook2/GenerateVersionInfo.ps1

```diff
@@ -0,0 +1,34 @@
+﻿$ErrorActionPreference = 'stop'
+
+Function Test-GitAvailable {
+    Try {
+        If (Get-Command git) {
+            return $true
+        }
+    }
+    Catch {
+        return $false
+    }
+}
+
+$buildDate = Get-Date -Format "yyyy-MM-dd HH:mm:ss K"
```

Comment:
Add `-AsUTC`?

Reply:
>Get-Date : A parameter cannot be found that matches parameter name 'AsUtc'.

Reply:
Needs newer powershell I guess?

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-date?view=powershell-7.1#example-10--return-a-date-value-interpreted-as-utc

Reply:
Probably. PowerShell Core adds a bunch of new stuff... but also removes some stuff. It's also not preinstalled with Windows so we can't rely on it existing.

Reply:
`[DateTime]::UtcNow.ToString('yyyy-MM-dd HH:mm:ss K')`

Reply:
tbh I’d rather keep it in the system timezone unless you can also figure out how to get the git commit time in utc. I guess we could parse and reformat it?

Reply:
Getting git commit time is pointless when you already have the commit.